content/docs/user-guide/networking/multiplayer/VersionMismatch.md

```diff
@@ -0,0 +1,36 @@
+---
+linkTitle: Version Mismatch
+title: Multiplayer Version Mismatch
+description: Learn how to detect multiplayer version mismatch in Open 3D Engine (O3DE).
+weight: 900
+---
+
+In order to keep the multiplayer simulation in sync, it's important that all connected multiplayer endpoints are running the same multiplayer version.
+
+For example, consider a server that is running a particular build of a multiplayer game. If an updated client connects with changes to its network components, the server may not know how to handle the updated network properties, or how to serialize network packets correctly.
+
+Servers and clients must be running the same version of all the multiplayer components (components which communicate to each other over the network); any differences may lead to unexpected behavior. 
+
+**Open 3D Engine (O3DE)** networking provides multiplayer version checks to identify and guard against this unexpected behavior.
```

Comment:
I think this is not much of an unexpected behavior as unsupported behavior. We do not support communication between two endpoints that have different versions of net components.

Reply:
I'd say communication is supported, but dangerous. Changing the name of one of the component properties, and everything will still work. Add a new property, all the other properties will still work, but the new property wont.